src/common/filter/Expressions.cpp

```diff
@@ -619,6 +621,19 @@ const char* FunctionCallExpression::decode(const char *pos, const char *end) {
     return pos;
 }
 
+std::string UUIDExpression::toString() const {
```

Comment:
Why we need UUIDExpression?  Why not implement it in FunctionManager?

Reply:
Because when `uuid` is called, some context should be referenced such as space ID and storage client.  FunctionManager is a singleton use Expression maybe better.